app/screens/reaction_list/reaction_row/reaction_row.js

```diff
@@ -11,20 +11,16 @@ import {
 } from 'react-native';
 import MaterialIcon from 'react-native-vector-icons/MaterialIcons';
 
-import {displayUsername} from '@mm-redux/utils/user_utils';
+import {getFullName} from '@mm-redux/utils/user_utils';
```

Comment:
why are we changing this? the display name should follow my preferences, right ?